engine/lib/upgrades/2015041400-1.11.0_dev-trim_metastrings-d9a9fdfa28a981a3.php

```diff
@@ -1,31 +0,0 @@
-<?php
-/**
- * Elgg 1.11.0-dev upgrade 2015041400
- * trim_metastrings
- *
- * Trim all whitespace padding in metastrings
- */
-
-$prefix = elgg_get_config('dbprefix');
-$q = "SELECT * FROM {$prefix}metastrings WHERE string REXEXP '^[ ]+|[ ]+$'";
-$r = mysql_query($q);
```

Comment:
Note I deleted these instead of figuring out how to do this cleanly. If we expose a result PDOStatement somehow we can do this.